pkg/scheduler/framework/interface.go

```diff
@@ -153,6 +153,12 @@ func NewPodsToActivate() *PodsToActivate {
 	return &PodsToActivate{Map: make(map[string]*v1.Pod)}
 }
 
+// A list of scored nodes, returned from scheduling.
+type SortedScoredNodes interface {
```

Comment:
All type/field comments should start with the type/field name, i.e.
```suggestion
// SortedScoredNodes is a list of scored nodes, returned from scheduling.
type SortedScoredNodes interface {
```
Please go through all changes and change the comments accordingly